From 9f401bf21707b484aeb2d2f6c1ed2ce41c50f253 Mon Sep 17 00:00:00 2001
From: Rob Browning <rlb@defaultvalue.org>
Date: Sun, 30 Jul 2017 21:43:10 -0500
Subject: Disable eieio-test-method-order-list-6

The eieio-test-method-order-list-6 test has been disabled

Its behavior is currently unpredictable.

Bug: https://debbugs.gnu.org/27878
Bug: https://debbugs.gnu.org/cgi/24503#21
---
 test/automated/eieio-test-methodinvoke.el | 20 ++++++++++----------
 1 file changed, 10 insertions(+), 10 deletions(-)

diff --git a/test/automated/eieio-test-methodinvoke.el b/test/automated/eieio-test-methodinvoke.el
index 09edea461d1..ab3a4444ee0 100644
--- a/test/automated/eieio-test-methodinvoke.el
+++ b/test/automated/eieio-test-methodinvoke.el
@@ -189,16 +189,16 @@ make-instance
   (cl-call-next-method)
   )
 
-(ert-deftest eieio-test-method-order-list-6 ()
-  (let ((eieio-test-method-order-list nil)
-	(ans '(
-	       (:STATIC C)
-	       (:STATIC C-base1)
-	       (:STATIC C-base2)
-	       )))
-    (C nil)
-    (setq eieio-test-method-order-list (nreverse eieio-test-method-order-list))
-    (eieio-test-match ans)))
+;; (ert-deftest eieio-test-method-order-list-6 ()
+;;   (let ((eieio-test-method-order-list nil)
+;; 	(ans '(
+;; 	       (:STATIC C)
+;; 	       (:STATIC C-base1)
+;; 	       (:STATIC C-base2)
+;; 	       )))
+;;     (C nil)
+;;     (setq eieio-test-method-order-list (nreverse eieio-test-method-order-list))
+;;     (eieio-test-match ans)))
 
 ;;; Diamond Test
 ;;
